### PR TITLE
[NEO] Implemented Enormous Energy Blade

### PR DESCRIPTION
**Pull request by craigbrennan24**

### CODE REVIEW

File: Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    public EnormousEnergyBlade copy() {
+        return new EnormousEnergyBlade(this);
+    }
+}
+
+class TapAttachedEffect extends TapEnchantedEffect {
+	
+	public TapAttachedEffect() {
+		super();
+		this.staticText = "tap that creature";
```
Comment (weirddan455) [inline Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java:50]: There's no need to make a class just to change text.  Just use `.setText()`

**Review by weirddan455 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java (hunk 1 above)
Comment (craigbrennan24) [reply Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java]: Sure, but I thought it helped with clarity of the effect. The TapEnchantedEffect class performed the needed effect but it didn't seem right using it with an equipment.

No worries though, I'll change it.

Comment (weirddan455) [reply Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java]: If you want to clean up that class, the best way would be to rename it to `TapAttachedEffect` and then have it take in the `AttachmentType` enum as a constructor argument.  Then use that to set the text accordingly.  See `AttacksIfAbleAttachedEffect` as an example.

For this card, it doesn't actually use the "equipped" verb so you would end up needing to set custom text regardless.

Also, when you write a custom class for a card, they're almost always named something like `EnormousEnergyBladeEffect` or something so it's clear that it's only used for that card (and not a generic class in use globally).

What you've got now looks good though so I'm happy to merge it as is.

**Review by weirddan455 — APPROVED** [review]

File: Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java (hunk 1 above)
Comment (craigbrennan24) [reply Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java]: Nice, I'll keep what you said in mind for the future though, thanks for the feedback!

**Pull request merged**